src/redux/slices/QuranReader/translations.ts

```diff
@@ -0,0 +1,22 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+const DEFAULT_TRANSLATIONS = ['20'];
+
+const initialState: string[] = DEFAULT_TRANSLATIONS;
+
+export const translationsSlice = createSlice({
+  name: 'translations',
+  initialState,
+  reducers: {
+    setCurrentTranslations: (state, action: PayloadAction<string[]>) => {
+      // this is to handle if the user un-selects all translations, we will reset it to defaults.
+      return action.payload.length ? action.payload : DEFAULT_TRANSLATIONS;
```

Comment:
I do not think that we should do this, if the user unselects all translations, we should show nothing, similar to the current experience on the live Quran.com

There are two reasons:
1) the experience will feel buggy, if a user unchecks something and then it is selected all of suddent
2) there are legitimate use cases for people not wanting to have a translation in the translation view. (e.g. a user primarily uses QDC in arabic, and uses the translation view to play audio of specific verses)